encrypted-media-respec.html

```diff
@@ -2028,9 +2028,29 @@
 
     <section>
       <h2><a>HTMLMediaElement</a> Extensions</h2>
-      <p>This section specifies additions to and modifications of the <a def-id="htmlmediaelement"></a> [[!HTML5]] when the Encrypted Media Extensions are supported.</p>
-
-      <p>When a <a def-id="htmlmediaelement"></a> is created, its internal <var>waiting for key</var> value is initialized to <code>false</code>.</p>
+      
+      <!-- Put all the monkey-patching here -->
+      
+      <p>This section specifies additions to and modifications of the <a def-id="htmlmediaelement"></a> [[!HTML5]] when the Encrypted Media Extensions are supported:</p>
```

Comment:
nit: This is a statement so it should end in a period. Since the list does not contain all modifications/additions (there are subsections), it doesn't make sense to reword to keep the colon.

When I suggested a list, I was referring to a list of the new variables. One option to address the nit and increase readability would be to have two separate lists - one of internal values and one of modifications to existing values and algorithms. I'm open to other options too.